src/components/entity-panel/sections/AccordionItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <v-expansion-panel class="custom-expansion-panel">
+    <v-expansion-panel-header class="custom-expansion-panel-header pl-3 pr-0 py-0" hide-actions>
+      <template v-slot:default="{ open: boolean }">
+        <v-icon :class="['custom-caret', (boolean) ? 'custom-caret-on' : '']" color="white" small>
+          fa-chevron-down
+        </v-icon>
+        <span class="pl-3">
+          <slot name="header">Header</slot>
+        </span>
+        <slot name="middleSpace"></slot>
+        <v-divider v-if="icon" vertical />
+          <v-btn
+            icon
+            class="custom-button"
+            :disabled="rightButtonDisabled"
+            @click.stop="$emit('rightButtonClicked')"
+          >
+            <v-icon color="white" small>
+              {{ icon }}
+            </v-icon>
+          </v-btn>
+      </template>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content class="custom-expansion-panel-content">
+      <slot name="content" />
+    </v-expansion-panel-content>
+  </v-expansion-panel>
+</template>
+
+<script lang="ts">
+
+import Component from 'vue-class-component'
+import Vue from 'vue'
+import { Prop } from 'vue-property-decorator'
+
+@Component({
+  name: 'AccordionItem'
+})
+export default class AccordionItem extends Vue {
+  @Prop() private readonly icon!: string
+  @Prop() private readonly rightButtonDisabled!: boolean
+}
+</script>
+<style lang="scss" scoped>
+.custom-button {
+  max-width: 50px;
+  justify-content: center;
+}
+
+.custom-expansion-panel {
+  border-left: 0.85px solid rgba(0, 0, 0, 0.12);
+}
+
+.custom-expansion-panel-header {
+  border: 1px solid rgba(0, 0, 0, 0.12);
+  background: $room-primary;
+  height: 35px !important;
```

Comment:
Im just curious why these need to be done via `!important` isn't there any other way?

Reply:
these are overriding Vuetify styles which have a high specificity, I will look at alternative ways

Reply:
I can actually remove all these important and the style is still applied :)